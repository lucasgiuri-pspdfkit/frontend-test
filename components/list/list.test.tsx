import List from "./list";
import { render } from "@testing-library/react";

const arrayWithOneItem = [
  {
    id: "1",
    name: "test",
    stars: 0,
    forks: 0,
    url: "https://github.com",
  },
];

const arrayWithTenItem = [
  {
    id: "1",
    name: "test",
    stars: 0,
    forks: 0,
    url: "https://github.com",
  },
  {
    id: "2",
    name: "test",
    stars: 0,
    forks: 0,
    url: "https://github.com",
  },
  {
    id: "3",

    name: "test",
    stars: 0,
    forks: 0,

    url: "https://github.com",
  },
  {
    id: "4",

    name: "test",
    stars: 0,

    forks: 0,
    url: "https://github.com",
  },
  {
    id: "5",

    name: "test",
    stars: 0,
    forks: 0,

    url: "https://github.com",
  },
  {
    id: "6",

    name: "test",
    stars: 0,
    forks: 0,

    url: "https://github.com",
  },
  {
    id: "7",

    name: "test",
    stars: 0,
    forks: 0,

    url: "https://github.com",
  },
  {
    id: "8",

    name: "test",
    stars: 0,

    forks: 0,

    url: "https://github.com",
  },
  {
    id: "9",

    name: "test",
    stars: 0,
    forks: 0,

    url: "https://github.com",
  },
  {
    id: "10",

    name: "test",
    stars: 0,
    forks: 0,

    url: "https://github.com",
  },
];

describe("List", () => {
  it("Snapshot renders with just one item", () => {
    const view = render(<List items={arrayWithOneItem} />);
    expect(view).toMatchSnapshot();
  });

  it("Snapshot renders with ten items", () => {
    const view = render(<List items={arrayWithTenItem} />);
    expect(view).toMatchSnapshot();
  });
});
