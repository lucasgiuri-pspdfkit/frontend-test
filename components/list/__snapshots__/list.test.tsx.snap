// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Snapshot renders with just one item 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="w-full h-full overflow-y-scroll"
      >
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="w-full h-full overflow-y-scroll"
    >
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`List Snapshot renders with ten items 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="w-full h-full overflow-y-scroll"
      >
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
        <li
          class="w-full py-4 border-b border-gray-300"
        >
          <a
            class="w-full flex items-center justify-between"
            href="https://github.com"
            target="_blank"
          >
            <h3
              class="text-lg font-bold text-gray-450"
            >
              test
            </h3>
            <div
              class="flex items-center"
            >
              <label
                class="flex items-center mr-2"
              >
                <img
                  alt="stars"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/star.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
              <label
                class="flex items-center"
              >
                <img
                  alt="forks"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="/icons/fork.svg"
                  style="color: transparent;"
                  width="16"
                />
                <span
                  class="pl-1"
                >
                  0
                </span>
              </label>
            </div>
          </a>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="w-full h-full overflow-y-scroll"
    >
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
      <li
        class="w-full py-4 border-b border-gray-300"
      >
        <a
          class="w-full flex items-center justify-between"
          href="https://github.com"
          target="_blank"
        >
          <h3
            class="text-lg font-bold text-gray-450"
          >
            test
          </h3>
          <div
            class="flex items-center"
          >
            <label
              class="flex items-center mr-2"
            >
              <img
                alt="stars"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/star.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
            <label
              class="flex items-center"
            >
              <img
                alt="forks"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/fork.svg"
                style="color: transparent;"
                width="16"
              />
              <span
                class="pl-1"
              >
                0
              </span>
            </label>
          </div>
        </a>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
